### DISCUSSION TITLE
Introduction - vxern

### DISCUSSION THREAD

**Discussion by vxern** [Tatoeba/tatoeba2#3077] [General]
Hi, I'm vxern, a software developer primarily working with TypeScript and Dart on the back-end. I speak Polish and English, with an interest in Romanian and Armenian.

My friend, @2colours, a fellow developer, introduced me Tatoeba some two years ago. We both admire the idea and motivation behind Tatoeba.

While looking for projects to work on together, a few weeks ago, the suggestion came that we could both start contributing to Tatoeba. Given our knowledge of writing and maintaining software, we could definitely be of some help.

Thus, here I am. Nice to be here.

**Comment by jiru** [comment]
Hello and welcome here both of you! :slightly_smiling_face:

Your message is a very a pleasant surprise. Any help is very much appreciated!

My (nick)name is gillux, I joined Tatoeba as a contributor a while ago. I mainly contribute on the backend, but I have came to touch pretty much every part of the code over the years. Maybe others will show up in the discussion, but as for "active" other developers there is also @trang, the kindest person I ever worked with, who created Tatoeba and is very busy with other matters at the moment, and maybe you could bump into @Yorwba who is always very helpful.

You probably already looked at our [contributing guide](https://github.com/Tatoeba/tatoeba2/wiki/Contributing-as-a-developer). Compared to Typescript and Dart, the technology Tatoeba is based on may feel a bit old school, but hey, it’s over 16 years old now, and still rocking! (Or so I hope :wink:)

Tatoeba’s development have been going through ups and downs, and at the moment it is rather on the down, or let’s say quiet time. Please take your time to install Tatoeba and to get familiar with the code as well as the community of language enthusiasts on https://tatoeba.org/. And if you have any question, feel free to ask here.

If you’d like, we can arrange a voice call to better introduce you with the project. Or continue with messages on this thread, it’s also fine.

**Reply by 2colours** [reply]
Hi @jiru 

we have narrowly met before I think. My username is Polgar1 on Tatoeba.

It took some time to return but maybe with @vxern we can actually get something done and persist.

I'm also not really a PHP expert (or a Vagrant expert for that matter, gotta set up some environment again) but hopefully I have enough clues about technology in general that I can just pick stuff up gradually.

Also, managing open-source projects seems like an interesting challenge, after a partially successful encounter with the Raku programming language. I also have very good memories of the way Tatoeba is managed so looking forward to getting some good work done!

I think it would be nice to have a voice call when it's good for you guys.

**Comment by trang** [comment]
Hi @vxern! Awesome to hear from you!

As @jiru mentioned, I am currently very busy with other matters. More specifically, doing one of the most difficult and exhausting jobs in the world — that is being a mom 😅 , so my involvement in Tatoeba has been very limited for the past 12 months and will probably remain very limited for at least the next 2-3 months. Therefore, I will sadly not be much around to guide you into your first contributions. But I still wanted to take the time to say welcome! And I'm sure @jiru can provide you with all the assistance you need.

Additionally, I wanted to point out that a while ago, we did some live stream with @2colours where he installed Tatoeba:
→ https://www.youtube.com/live/2NYkhE2y2bk?si=TWllzp9HLRah7aMD
and learned a bit about how to contribute to the API:
→ https://www.youtube.com/live/PLu5bPiIGPw?si=ol_DZbue0oyTO_ng
Some info in there might be a bit outdated and also, I didn't edit the videos to shorten them to the essential parts, so you'll probably have to use the x2 speed and fast forward quite a lot, but perhaps it helps a bit to watch them so you can have a more concrete idea of how to get started and how to navigate into the code 🙂 

And @2colours, very glad you returned and brought your friend along!

Enjoy your time here guys! 🤗
